ver, crear, modificar y eliminar ofertas + cálculo correcto en web

corp-0 opened the pull request:
fixes #167 
Es necesario migrar para probar los cambios y una vez se haya mergeado.

ver commits para lista ordenada de cambios.

corp-0 committed:
chore: migraciones

corp-0 committed:
fix: ya no es posible agregar cantidad de producto mayor a stock

corp-0 committed:
refactor: simplifica query de ofertas y precio final de producto

corp-0 committed:
feat: agregado ver, crear, modificar y eliminar ofertas

kanroot commented on src/inventario/forms.py:
```diff
@@ -26,3 +27,25 @@ class ActualizarProductoVentasForm(forms.ModelForm):
         fields = ['es_nuevo', 'es_destacado', 'esta_publicado', 'precio']
+
+
+class OfertaForm(forms.ModelForm):
```
Para crear una oferta debes seleccionar la fecha, el picker te esta modificando el mes y no el día.

ejemplo: quiero el 24/06/2022

me deja el campo 06/24/2022

corp-0 replied:
Voy a revisarlo

corp-0 replied:
corregido

corp-0 committed:
fix: corrige error del datepicker

La vista de editar no tenía especificado el form que da el formato a la fecha, por lo que intentaba modificar el mes en lugar del día al tomar un valor del datepicker.
